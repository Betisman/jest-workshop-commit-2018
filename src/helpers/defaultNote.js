
export default () => ({
  title: 'write your note title :)',
  description: 'you can write lorem ipsum dolor ¯\_(ツ)_/¯',
});