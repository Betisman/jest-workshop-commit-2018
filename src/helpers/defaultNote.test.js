import defaultNote from './defaultNote';

it('returns a object content', () => {
  expect(defaultNote()).toMatchSnapshot();
});
