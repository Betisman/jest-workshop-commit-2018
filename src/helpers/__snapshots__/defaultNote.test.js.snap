// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns a object content 1`] = `
Object {
  "description": "you can write lorem ipsum dolor ¯_(ツ)_/¯",
  "title": "write your note title :)",
}
`;
